readme.md

```diff
@@ -255,18 +255,22 @@ You can use the [`oauth-1.0a`](https://github.com/ddo/oauth-1.0a) module to crea
 
 ```js
 const got = require('got');
+const crypto  = require('crypto');
 const OAuth = require('oauth-1.0a');
 
 const oauth = OAuth({
 	consumer: {
-		public: process.env.CONSUMER_KEY,
+		key: process.env.CONSUMER_KEY,
 		secret: process.env.CONSUMER_SECRET
 	},
-	signature_method: 'HMAC-SHA1'
+	signature_method: 'HMAC-SHA1',
+	hash_function: (base_string, key) => {
+		return crypto.createHmac('sha1', key).update(base_string).digest('base64');
```

Comment:
like that: `hash_function: (base_string, key) => crypto.createHmac('sha1', key).update(base_string).digest('base64')` ? I'm ok with, but it's slightly harder to read or not?